source/docs/b1.vectors.md

```diff
@@ -1,9 +1,9 @@
 title: Vectors
 ---
 
-If you remember, array is a  fixed-size list of elements, of same data type. Even with mut, it’s element count can not be changed. A vector is kind of **a re-sizable array** but **all elements must be in the same type**.
+If you remember, array is a  fixed-size list of elements, of same data type. Even with mut, its element count can not be changed. A vector is kind of **a re-sizable array** but **all elements must be in the same type**.
 
-⭐️ It’s a generic type, written as **Vec<T>** . T can have any type, ex. The type of a Vec of i32s is Vec<i32>. Also Vectors always allocate their data in dynamically allocated heap.
+⭐️ It’s a generic type, written as **Vec\<T>** . T can have any type, ex. The type of a Vec of i32s is Vec\<i32>. Also Vectors always allocate their data in dynamically allocated heap.
```

Comment:
why adding \

Reply:
If not, then the `<bla>` is filtered as an HTML tag and it is not visible... You don't have the problem when using the \`\<bla>\`

Reply:
ah, just noticed it. nice catch. Thanks 👍

Reply:
There may be some other occurrences elsewhere... But it was making the text a bit harder te understand :D
Will update the other part of the PR later today !

Reply:
Thanks